### PR TITLE
v8.6.0 changelog and README updated

### PR DESCRIPTION
**Pull request by jagadeeshnv**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,45 @@ Dell OpenManage Ansible Modules Release Notes
...
 
 
+v8.6.0
+======
+
+Release Summary
+---------------
+
+- Support for environment variables as fallback for credentials for all modules.
+- Enhanced idrac_certificates to support `CUSTOMCERTIFICATE` and upload of SSL key.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- Enhanced idrac_certificates module and idrac_certificate role to support `CUSTOMCERTIFICATE` and `HTTPS` certificate import with SSL key.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,45 @@ Dell OpenManage Ansible Modules Release Notes
...
+
+Major Changes
+-------------
+
+- All OME modules are enhanced to support environment variables `OME_USERNAME` and `OME_PASSWORD` as fallback for credentials.
+- All iDRAC and redfish modules are enhanced to support environment variables `IDRAC_USERNAME` and `IDRAC_PASSWORD` as fallback for credentials.
+- idrac_certificates - The module is enhanced to support import and export of `CUSTOMCERTIFICATE`.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_certificates module and idrac_certificate role - This is enhanced to support import and export of `CUSTOMCERTIFICATE`.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,45 @@ Dell OpenManage Ansible Modules Release Notes
...
+
+Minor Changes
+-------------
+
+- idrac_certificates - The `email_address` is no longer a required parameter.
+- idrac_certificates - The module and role are enhanced to support upload of SSL key along with `HTTPS` certificate.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_certificates - The module and role are enhanced to support import of `HTTPS` certificate with SSL key.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,45 @@ Dell OpenManage Ansible Modules Release Notes
...
+-------------
+
+- All OME modules are enhanced to support environment variables `OME_USERNAME` and `OME_PASSWORD` as fallback for credentials.
+- All iDRAC and redfish modules are enhanced to support environment variables `IDRAC_USERNAME` and `IDRAC_PASSWORD` as fallback for credentials.
+- idrac_certificates - The module is enhanced to support import and export of `CUSTOMCERTIFICATE`.
+
+Minor Changes
+-------------
+
+- idrac_certificates - The `email_address` is no longer a required parameter.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_certificates module and idrac_certificate role - The `email_address` has been made an optional parameter.
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,6 +5,45 @@ Dell OpenManage Ansible Modules Release Notes
...
+- idrac_certificates - The module and role are enhanced to support upload of SSL key along with `HTTPS` certificate.
+
+Bugfixes
+--------
+
+- Fixed an issue where the `NO_PROXY` environment variable was being ignored. (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/554)
+- idrac_certificates - The `email_address` is no longer a required parameter. (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/582).
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_certificates - The `email_address` has been made an optional parameter. (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/582).
```

**Review by rajshekarp87 — CHANGES_REQUESTED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by Kritika-Bhateja-03 — APPROVED** [review]

**Pull request merged**